src/shared/lib/NumberUtils.ts

```diff
@@ -5,6 +5,8 @@ export function toConditionalPrecision(
 ): string {
     if (0.000001 <= number && number < threshold) {
         return number.toExponential(precision);
+    } else if (number === 0) {
+        return '< 10^-10';
```

Comment:
Not sure if this is the right place to add this condition. `toConditionalPrecision` seems like a general purpose utility function used in multiple places, not only in the comparison page. I would add this check directly in the comparison page to make sure that we don't accidentally break other components.

Reply:
@aswin2108 Did you try using the table value directly? Looks like we have an already working implementation there. Why not just do the same thing here? https://github.com/cBioPortal/cbioportal-frontend/blob/099e4e9f3aba66bf83b64167d81d7be29b002d0a/src/pages/resultsView/survival/SurvivalPrefixTable.tsx#L222